pkg/operator/ceph/nfs/nfs.go

```diff
@@ -55,7 +55,8 @@ type daemonConfig struct {
 
 // Create the ganesha server
 func (r *ReconcileCephNFS) upCephNFS(n *cephv1.CephNFS) error {
-	nfsToSkipReconcile, err := controller.GetDaemonsToSkipReconcile(r.clusterInfo.Context, r.context, n.Namespace, config.NfsType, AppName)
+	instanceLabelKey := "instance"
```

Comment:
I was referring to a global const in this file. That can also be used in your test files.

Reply:
Okay, test file is in different package too